src/app/documentation-messages.service.ts

```diff
@@ -0,0 +1,69 @@
+import { Injectable } from '@angular/core';
+import { HttpClient, HttpErrorResponse } from '@angular/common/http';
+import { NotificationService, NotificationType } from 'patternfly-ng';
+import { BehaviorSubject, Observable } from 'rxjs';
+import { filter, map } from 'rxjs/operators';
+
+@Injectable({
+  providedIn: 'root'
```

Comment:
Neat, I hadn't seen this feature before. Probably worth updating the other services to do the same thing. I'll create an issue for it.